Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -621,9 +624,19 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
                 // Add to document map
                 documentLocationMap[url] = technologyResult.topicGraphNode.reference
                 
-                let sourceLanguage = SourceLanguage.swift
-                let node = DocumentationNode(reference: technologyResult.topicGraphNode.reference, kind: .technology, sourceLanguage: sourceLanguage, name: .conceptual(title: technology.intro.title), markup: technology.originalMarkup, semantic: technology)
-                documentationCache[technologyResult.topicGraphNode.reference] = node
+                let reference = technologyResult.topicGraphNode.reference.withSourceLanguages(sourceLanguages)
+                
+                let node = DocumentationNode(
```

Comment:
Readability nit:
```suggestion
                let technologyNode = DocumentationNode(
```

Probably partly due to the GitHub UI but this diff was generally hard for me to follow because we have a lot of otherwise unspecified `node`s and `reference`.

Reply:
Makes sense, I see it's hard to follow in the GitHub diff as well. I'll take your suggestions!